### DISCUSSION TITLE
Vercel, Inc. and EU-U.S. Privacy Shield Framework

### DISCUSSION THREAD

**Discussion by steveDvlpr** [vercel/next.js#58385] [Help]
### Summary

I'm working on a new project that is built with next.js, but we are located within the EU. The problem is, that we are basically only "allowed" to use a US service provider if the provider complies with the EU-U.S. Privacy Shield Framework.

In the Vercel Privacy Policy, they write the following:

> Vercel complies with the EU-U.S. and Swiss-U.S. Privacy Shield framework ("Privacy Shield") as set forth by the U.S. Department of Commerce in connection with the collection, use, and retention of certain personal data transferred from the EEA or the UK and Switzerland to the United States. [...] To learn more about the Privacy Shield program, or to view our certification, please see https://www.privacyshield.gov/.

The problem is, that on privacyshield.gov they are marked as ***invalid***. Is there any chance this will be renewed? I'd really love to use Vercel to deploy my projects, but if I can get in trouble because I'm in the EU, I will have to switch to a different provider.

Thanks!

### Additional information

```js
https://vercel.com/legal/privacy-policy#international-transfers
```

**Comment by g-rasse** [comment]
+1

**Comment by timohausmann** [comment]
It seems Vercel is not a valid hosting option for projects located in Germany/EU, sadly.

Here is the invalid profile https://www.privacyshield.gov/ps/participant?id=a2zt0000000TTIbAAO

**Comment by g-rasse** [comment]
Just got a new email from Vercel about an update to the Privacy Policy ( [Update] Vercel Privacy Notice and New Subprocessors )

> We have updated our [Privacy Notice](https://vercel.com/legal/privacy-policy) to reflect new feature developments and product expansion.
> 
> Notable updates include:
> 
>
> -  **Updates to our data protection compliance and security practices, including the Data Privacy Framework certification.**
> - More details on jursidiction-specific information based on regulatory developments, including new data subject rights and enhanced processes for exercising those rights.
> - Improved readability and clarity around Vercel’s services, the information collected, and how it is disclosed.
> ...

Reading this I thought the invalid profile would be fixed / updated.
Reading through the privacy policy it still says this:

> ...
Vercel complies with the EU-U.S. Data Privacy Framework (EU-U.S. DPF), the UK Extension to the EU-U.S. DPF, and, upon entering into force, the Swiss-U.S. 
...
**To learn more about the Data Privacy Framework (DPF) program, and to view our certification, please visit the [Data Privacy Framework website](https://www.dataprivacyframework.gov/).**
...

However when actually viewing the link and going to the [Vercel profile](https://www.dataprivacyframework.gov/list), it still says inactive.

**Comment by timohausmann** [comment]
I reached out to Vercel after they send this e-mail out and today they wrote me this:

> The Data Privacy Framework team just updated their active participants list. Vercel's profile is available at https://www.dataprivacyframework.gov/list.

There is no deep link but searching for Vercel shows: 
EU-U.S. Data Privacy Framework : 🟢 Active

🎉

Edit: They also say privacyshield.gov (that I linked to above) is a separate website and framework.
